tests/format/typescript/module/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,6 +117,10 @@ namespace X {
   }
 }
 
+namespace /* module */ X {}
+module /* namespace */ X {}
+module /* namespace */ "x" {}
```

Comment:
Only happens with `parser: 'babel-ts'`, not going to add changelog for it.

**Prettier 3.2.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEUCGBbOBnADmsOAAgHoAqIjCAEwFcAbYskogDSOAF8AdKKuxqQroseAkxbsuIADQgIuGAEto2ZKDQAnTRADuABS0I1KNPV1oAnmrkAjTQQDWcGAGVMcADJKocZADMzbDg7BzBnN3wwHwBzZBhNWhCQYIwleMTkuAAPXDhNJSxYMwAVfPQCnACg5OxYxgBFWgh4avpguQArbGzXergmlr8kQPbkgEdm+H0dXBMQNGwAWl84ajXZEAS0JXpYgGEIDAw0ZAX6ek26qBjGAEEYBKVbWmn87182jpAACxgMegAdR+SngYkIrmMoKUADdQZYzmBsDYQDCkgBJKDrWCuMAFRR3LGuGCWRhfZK4HTBQEOXBnSk4fIwvxyHzBTQwGZoGInclyfCadlnWxoWxwehLGAoyk+GCApTUGA-ZAADgADHJNHBJkotVyeacRjU5DBRfLFcqkAAmOS0YIlUUmUbfOAYMXUdbUTxoG60blwABiEE0J0esTOaFeEBAnE4QA)
<!-- prettier-ignore -->
```sh
--parser babel-ts
```

**Input:**
<!-- prettier-ignore -->
```tsx
namespace /* module */ X {}
module /* namespace */ X {}
```

**Output:**
<!-- prettier-ignore -->
```tsx
module /* module */ X {}
module /* namespace */ X {}

```